python/ray/dag/tests/experimental/test_torch_tensor_dag.py

```diff
@@ -407,6 +426,30 @@ def test_torch_tensor_nccl_nested_dynamic(ray_start_regular):
     compiled_dag.teardown()
 
 
+@pytest.mark.parametrize("ray_start_regular", [{"num_cpus": 4}], indirect=True)
+def test_torch_tensor_nccl_within_same_actor(ray_start_regular):
+    workers = [TrainWorker.remote() for _ in range(2)]
+
+    with InputNode() as inp:
```

Comment:
can we just make a test simple actually? just do 

```python3
entrypoint = workers[0].entrypoint.bind(inp)
entrypoint. with_type_hint(TorchTensorType(transport=TorchTensorType.NCCL))
dag = [worker.forward.bind(entrypoint) for worker in workers]
```

Reply:
+1

Reply:
I simplified it even further.